### ISSUE TITLE
`valid-prop-names-in-kit-pages` false positive when exporting `trailingSlash` variable

### ISSUE THREAD

**Issue by DetachHead** [ota-meshi/eslint-plugin-svelte#319]
### Before You File a Bug Report Please Confirm You Have Done The Following...

- [X] I have tried restarting my IDE and the issue persists.
- [X] I have updated to the latest version of the packages.

### What version of ESLint are you using?

8.28.0

### What version of `eslint-plugin-svelte` are you using?

2.14.0

### What did you do?

<details>
<summary>Configuration</summary>

```js
module.exports = {
    rules: {
        'svelte/valid-prop-names-in-kit-pages': 'error'
    }
}
```
</details>

```svelte
<!-- +page.svelte -->
<script lang="ts">
    export const trailingSlash = 'always'
</script>
```

### What did you expect to happen?

no error, as `trailingSlash` is valid to export on sveltekit page components. see https://github.com/sveltejs/kit/pull/7719

### What actually happened?

```
> eslint .


C:\projects\project\src\routes\+page.svelte
  10:18  error  disallow props other than data or errors in Svelte Kit page components  svelte/valid-prop-names-in-kit-pages
```

### Link to **GitHub Repo** with Minimal Reproducible Example

cant be bothered but i'm gonna make a PR fixing it myself anyway

**Comment by DetachHead** [comment]
nvm i misread the docs, it's only valid in `+layout.ts`, not `.svelte` files
